doc/Architecture/spec-al_x-road_audit_log_events.md

```diff
@@ -0,0 +1,465 @@
+| Import certificate from token                                                                                                              | <ul><li>tokenId - identifier of the token where imported certificate located</li><li>tokenSerialNumber - the serial number of the token</li><li>tokenFriendlyName - the friendly name of the token</li><li>keyId - the identifier of the key related to the imported certificate</li><li>keyFriendlyName - the friendly name of the key</li><li>keyUsage - the key usage</li><li>certId - the identifier of the imported certificate</li><li>certHash - the hash of the imported certificate</li><li>certHashAlgorithm - the hash algorithm used to calculate value of the field certHash</li><li>clientIdentifier - the client identifier of the member constructed from signing certificate</li></ul>                                                                                                                                                                                                       |
+| Delete certificate <br><br> Event name is used if event fails before it is known where certificate is delete from (configuration or token) |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
```

Comment:
Is the "Data fields" column empty on purpose?

Reply:
Yes, it's the same way in the original docx for the `Delete Certificate` field.